src/wrapper.test.js

```diff
@@ -1,5 +1,40 @@
 import * as fs from 'fs';
-import * as utils from './utils';
+
+/*
+ * Monkey-patch jest to be able to handle async in isolateModules
+ *
+ * See https://github.com/facebook/jest/issues/10428
+ */
+
+async function isolateModulesAsync(fn) {
```

Comment:
Nice workaround! can we put it in utils?

Reply:
Feel free to, but not in this PR.